rest/routes/candidate.py

```diff
@@ -0,0 +1,44 @@
+"""
+Endpoints connected with candidates
+"""
+import logging
+
+from flask import Blueprint, request
+from rest.common.response import create_response
+from rest.common.token import handle_request_token
+from rest.models.candidate import Candidate
+
+candidate_url = Blueprint('candidate', __name__)
+
+
+@candidate_url.route('/<candidate_pesel>', methods=['GET'])
+def get_candidate(candidate_pesel):
+    """
+    Endpoint returns all available information about specific candidate
+    :returns: flask response containing json with candidate data
+    """
+    logging.info("Getting candidate data")
+
+    role, response = handle_request_token(request)
+
+    if role is None:
+        logging.warning("Role is not specified!")
+        return response
+
+    if role != 'editor' or role != 'admin':
```

Comment:
Why only editor has access to candidate data?

Reply:
who shall have access to candidate data then?

Reply:
imho it is sensitive data and not all the users should have access to it